pwiz_tools/Skyline/Executables/BuildMethod/BuildSciexMethod/Program.cs

```diff
@@ -76,12 +80,14 @@ private static void Usage()
                 "   Takes template method file and a Skyline generated \n" +
                 "   transition list as inputs, to generate a new method file\n" +
                 "   as output.\n" +
+                "   -n               Extract device name from the template" + 
```

Comment:
No, I probably should remove this command. I added it. The idea was to show the instrument name for the selected template in the MethodExportDialog, but since SCIEX OS API fails for any method that doesn't match the currently active instrument it wouldn't be very useful.